foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiAssistantMessage.java

```diff
@@ -0,0 +1,31 @@
+package com.sap.ai.sdk.foundationmodels.openai;
+
+import com.sap.ai.sdk.foundationmodels.openai.model2.ChatCompletionRequestAssistantMessage;
+import com.sap.ai.sdk.foundationmodels.openai.model2.ChatCompletionRequestAssistantMessageContent;
+import javax.annotation.Nonnull;
+import lombok.Value;
+import lombok.experimental.Accessors;
+
+/** Represents a chat message as 'assistant' to OpenAI service. */
+@Value
+@Accessors(fluent = true)
+class OpenAiAssistantMessage implements OpenAiMessage {
+
+  /** The role of the message. */
+  @Nonnull String role = "assistant";
+
+  /** The content of the message. */
+  @Nonnull String content;
+
+  /**
+   * Converts the message to a serializable object.
+   *
+   * @return the corresponding {@code ChatCompletionRequestAssistantMessage} object.
+   */
+  @Nonnull
+  public ChatCompletionRequestAssistantMessage createDTO() {
```

Comment:
(Comment/Minor)

I understand you are mirroring the Orchestration API design, as suggested. However, I would like to challenge this method (visibility) here. We don't really want our convenience-users to access this method. While I can't challenge it in orchestration anymore (due to stability), maybe we can explore other options here in the new API. 

Can be done in a separate PR / ticket.